docs/_docs/contributing.md

```diff
@@ -125,19 +125,27 @@ site or configuration. [Feel free to check it out!](https://github.com/jekyll/je
 
 To run the test suite and build the gem you'll need to install Jekyll's dependencies by running the following command:
 
-<pre class="highlight"><code>$ script/bootstrap</code></pre>
+```sh
+script/bootstrap
+```
```

Comment:
I believe this page is generated, so this will have to be updated in `.github/CONTRIBUTING.md`. I may be wrong though!